frontend/server/src/main/java/org/pytorch/serve/job/RestJob.java

```diff
@@ -102,29 +63,33 @@ public void response(
          */
         if (ctx != null) {
             MetricAggregator.handleInferenceMetric(
-                    modelName, modelVersion, scheduled - begin, inferTime);
+                    getModelName(), getModelVersion(), getScheduled() - getBegin(), inferTime);
             NettyUtils.sendHttpResponse(ctx, resp, true);
         }
         logger.debug(
                 "Waiting time ns: {}, Backend time ns: {}",
-                scheduled - begin,
-                System.nanoTime() - scheduled);
+                getScheduled() - getBegin(),
+                System.nanoTime() - getScheduled());
     }
 
-    public void sendError(HttpResponseStatus status, String error) {
+    @Override
+    public void sendError(int status, String error) {
         /*
          * We can load the models based on the configuration file.Since this Job is
          * not driven by the external connections, we could have a empty context for
          * this job. We shouldn't try to send a response to ctx if this is not triggered
          * by external clients.
          */
         if (ctx != null) {
-            NettyUtils.sendError(ctx, status, new InternalServerException(error));
+            // Mapping HTTPURLConnection's HTTP_ENTITY_TOO_LARGE to Netty's INSUFFICIENT_STORAGE
+            status = (status == 413) ? 507 : status;
```

Comment:
This was because we refactored all the error codes sent from the `WLM` package to use HTTP Status code  Java's inbuilt `HTTPURLConnectio`n instead of Netty's `HTTPResponseStatus`. And `HTTPURLConnection` doesn't has the `507` code available. Hence, we used the `413` code instead and re-mapped it to `507` to keep the error codes backward compatible.